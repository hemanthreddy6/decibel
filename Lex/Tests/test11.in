audio heavenly_chorus <- load "track.mp3"; //Loading a track

audio sped_up_chaotic_chorus <- heavenly_chorus >> 2; //Speeding up the track
audio slowed_down_boring_chorus <- heavenly_chorus << 3; //Slowing down the track

//Playing the modified tracks
play sped_up_chaotic_chorus;
play slowed_down_boring_chorus;