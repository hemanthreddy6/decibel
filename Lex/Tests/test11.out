AUDIO
IDENTIFIER
<-
LOAD
STRING
;
AUDIO
IDENTIFIER
<-
IDENTIFIER
>>
NUMBER
;
AUDIO
IDENTIFIER
<-
IDENTIFIER
<<
NUMBER
;
PLAY
IDENTIFIER
;
PLAY
IDENTIFIER
;
